core/frontend/src/components/video-manager/VideoStreamCreationDialog.vue

```diff
@@ -281,14 +303,35 @@ export default Vue.extend({
     addNewEndpoint() {
       this.validateForm()
       this.stream_endpoints.push('')
+      this.set_default_address_for_stream(this.stream_endpoints.length - 1, StreamType.UDP)
+      this.validateForm()
     },
     removeEndpoint(index: number) {
       this.validateForm()
       this.stream_endpoints.splice(index, 1)
+      this.validateForm()
     },
     showDialog(state: boolean) {
       this.$emit('visibilityChange', state)
     },
+    set_default_address_for_stream(index: number, stream_type: StreamType) {
+      switch (stream_type) {
+        case StreamType.UDP:
+          if (!this.stream_endpoints[index].includes('udp://')) {
+            // Vue.set() forces the update of a nested property
+            Vue.set(this.stream_endpoints, index, `udp://${this.user_ip_address}:5600`)
+          }
+          break
+        case StreamType.RTSP:
+          if (!this.stream_endpoints[index].includes('rtsp://')) {
+            // Vue.set() forces the update of a nested property
+            Vue.set(this.stream_endpoints, index, `rtsp://${this.vehicle_ip_address}:8554/video0`)
```

Comment:
I wonder if `video0` could be `video{i}` where `i` automatically increases by the number of RTSP streams or perhaps anything unique because it will, in the best scenario, only work the first time.

Reply:
yep! index sounds nice, I'll add this to the udp ones too
I was thinking about checking for ports in use in udp, but this is a lot more straight forward and can be done in no new lines, nice!

Reply:
![image](https://user-images.githubusercontent.com/4013804/186743773-b622b49e-6a40-4074-aece-f45de5d20f84.png)